android/src/main/java/com/swmansion/rnscreens/ScreenFragment.java

```diff
@@ -238,6 +305,14 @@ public void run() {
     }
   }
 
+  public boolean isSendingProgress() {
+    return mIsSendingProgress;
+  }
+
+  public void setIsSendingProgress(boolean isSendingProgress) {
```

Comment:
typically when getter starts with "is" the setter does not need "is" in java (that's the getter/setter naming convention), so this should just be "setSendingProgress"